commons/ui/src/main/kotlin/com/teacherworkout/commons/ui/composables/LessonCard.kt

```diff
@@ -59,7 +68,7 @@ private fun LessonCardImage(imageResourceId: Int, contentDescription: String) {
 
 @Composable
 private fun LessonCardBody(lesson: Lesson) {
-    val space8dp = dimensionResource(id = R.dimen.space_8dp)
+    val space8dp = dimensionResource(id = com.teacherworkout.commons.ui.R.dimen.space_8dp)
```

Comment:
This looks pretty ugly, isn't there a way to use this only by name instead of the entire import path?

Reply:
Indeed it's ugly. We cannot do a file import because it clashes with the other R classes used in the same file. I'm not sure exactly what changed in the dependencies that makes the previous code not compile, probably the Android Gradle Plugin. But now that you mention it, I could try to use a Kotlin import alias to make this prettier.